docker.mk

```diff
@@ -1,6 +1,6 @@
 
 suse:
-	docker build -t 389-ds-suse:master -f docker/389-ds-suse/Dockerfile .
```

Comment:
Here it's a tag that IIRC corresponds to a branch name. There is a special tag `latest` that I think would make more sense here.

Reply:
Ahh I was waiting for a container expert to comment on this :-)  I'll get this fixed

Reply:
latest makes more sense, that's okay :)

Reply:
Since we renamed master to main in our repo, should this "main" now?

Reply:
> Since we renamed master to main in our repo, should this "main" now?

@vashirov @Firstyear I am going to change the tag to "main" first thing tomorrow, so if there are any objections let me know sooner than later :-)

Reply:
No problems here.

Reply:
> Since we renamed master to main in our repo, should this "main" now?

`latest` is fine, thanks!